packages/a2ui/tests/gea-loader.ts

```
@@ -0,0 +1,67 @@
+}
+const rawTransform = plugin.transform as
+  | ((code: string, id: string) => { code: string } | string | null | undefined)
+  | { handler: (code: string, id: string) => { code: string } | string | null | undefined }
+const transform = typeof rawTransform === 'function' ? rawTransform : rawTransform.handler
+
+const HERE = dirname(fileURLToPath(import.meta.url))
+
+const VIRTUAL_MODULES: Record<string, string> = {
+  'virtual:gea-compiler-runtime': pathToFileURL(
+    resolve(HERE, '../../gea/src/compiler-runtime.ts'),
+  ).href,
```

ESLint

[error] Replace `⏎····resolve(HERE,·'../../gea/src/compiler-runtime.ts'),⏎··` with `resolve(HERE,·'../../gea/src/compiler-runtime.ts')`

(prettier/prettier)